src/lib/controllers/ui-controller.js

```diff
@@ -1336,18 +1319,11 @@ export default class UIController {
     this.store.commit('app/setCurrentLanguage', currentLanguageID)
     const newLanguageCode = LanguageModelFactory.getLanguageCodeFromId(currentLanguageID)
     if (this.state.currentLanguage !== newLanguageCode) {
-      this.store.commit('app/resetGrammarData')
```

Comment:
I think we still want the currently expanded grammar to be in sync with the language of the lookup.

Reply:
We don't need resetGrammarData, because for now we have only the following actions with grammar:
   - update updatedGrammar flag and upload data to Vuex grammarData
And in a separate workflow - inside grammar tab we define what languge should be shown (it wasn't ready yesterday - but now it is ready) 

It looks the following way:
we have two properties inside component - `currentLanguageCode` and `centralLanguageCode`

`currentLanguageCode` - shows languageCode for showing grammar
`centralLanguageCode` - shows languageCode from UIController

every time `this.$store.state.app.currentLanguageCode` changes - it updates `centralLanguageCode` (lookup, change language in preferances)
if at this moment
   - `currentLanguageCode` is empty (no language is shown)
   or
   - grammar panel is invisible 
then `currentLanguageCode` becomes equal to `centralLanguageCode`
and this grammar is shown